doc/04-Upgrading.md

```diff
@@ -3,6 +3,54 @@
 Specific version upgrades are described below. Please note that version upgrades are incremental.
 If you are upgrading across multiple versions, make sure to follow the steps for each of them.
 
+## Upgrading to Icinga DB v1.1.2
+
+Please apply the `1.1.2.sql` upgrade script to your database. For package installations, you can find this file at
+`/usr/share/icingadb/schema/mysql/upgrades/` or `/usr/share/icingadb/schema/pgsql/upgrades/`, depending on your
+database vendor.
+
+!!! warning
+
+    In order to fix a loading performance issue of the `icingadb/history` view, this upgrade script adds an appropriate
+    index on the `history` table. However, this may hang for a relatively considerable time and block all Icinga DB
+    queries on all `*_history` tables and the `history` table inclusively if there is an ongoing, long-running query
+    on the `history` table. One way of causing this to happen is if an Icinga Web user accesses the `icingadb/history`
```

Comment:
I'd add a brief sentence in between here stating the desired behavior, something like: "Creating this index normally happens without blocking other queries. However, [there is this specific situation where some locking can go wrong ...]".

Reply:
Oops, by putting it into square brackets, I wanted to signal it as a sloppy summary of what is about to follow, not a word by word suggestion :joy:

Reply:
I will change that!